src/logging.js

```diff
@@ -1,41 +1,49 @@
+// @flow
+
 /**
  * Logging setup for our application.
  */
-const stream = require("stream");
-const expressWinston = require("express-winston");
-const winston = require("winston");
+import stream from "stream";
+import expressWinston from "express-winston";
+import winston from "winston";
+
+import {LoggingWinston as StackdriverTransport} from "@google-cloud/logging-winston";
 
-const StackdriverTransport = require("@google-cloud/logging-winston")
-    .LoggingWinston;
-const args = require("./arguments.js");
+import type {NpmLogLevels, Format} from "winston";
```

Comment:
The various imports here are a bit hard to read - might be a bit easier to keep all the type imports together.

Reply:
Yeah, I agree, there's an eslint or prettier rule (copied from WB, I think) that's rearranging them. I will find which one it is and oust it.

Reply:
I tracked this down to the `import/order` rule. This issue in Github is tracking a change to the rule to make it put the type imports after the regular imports: https://github.com/benmosher/eslint-plugin-import/issues/645

For now, I'm going to remove this rule from our linter setup.